create.php

```diff
@@ -1,15 +1,64 @@
 <?php
+// The page was accessed directly, bail...
+if (
+	!isset($_SERVER['HTTP_X_REQUESTED_WITH'])
+	|| strtolower($_SERVER['HTTP_X_REQUESTED_WITH']) !== 'xmlhttprequest'
+) {
```

Comment:
Add validation of a CSRF token?

Reply:
As mentioned in the PR comment, I didn't do anything security related (that I would normally do in real life). Happy to add it at a later stage if you like